playbooks/cluster_setup_auditd.yaml

```diff
@@ -8,10 +8,25 @@
   tasks:
   - name: copy syslog.conf
     copy:
-      src: ../src/debian/syslog.conf
+      src: ../src/debian/auditd/syslog.conf
       dest: /etc/audit/plugins.d/syslog.conf
 
   - name: copy audit.rules
     copy:
-      src: ../src/debian/audit.rules
+      src: ../src/debian/auditd/audit.rules
       dest: /etc/audit/rules.d/audit.rules
+
+  - name: copy python ioperm
```

Comment:
As stated in a previous comment, I think this should be moved to cukinia-tests

Reply:
Done: https://github.com/seapath/cukinia-tests/pull/30/commits/ab895ce098ebfecdd8787858b7bdd189303797ea